uaclient/status.py

```diff
@@ -172,9 +172,9 @@ class UserFacingStatus(enum.Enum):
 To use '{name}' you need an Ubuntu Advantage subscription.
 Personal and community subscriptions are available at no charge
 See https://ubuntu.com/advantage"""
-MESSAGE_ENABLE_BY_DEFAULT_TMPL = "Enabling default service '{name}'"
+MESSAGE_ENABLE_BY_DEFAULT_TMPL = "Enabling default service {name}"
 MESSAGE_ENABLE_BY_DEFAULT_MANUAL_TMPL = """\
-Service '{name}' is recommended by default. To enable run: ua enable {name}"""
+Service {name} is recommended by default. Run: sudo ua enable {name}"""
```

Comment:
I don't see this message in the UX doc at all. @powersj, this is the message we emit when `--no-auto-enable` is passed to `ua attach`, can you confirm that this is what we want?

Reply:
+1